### PR TITLE
add GetString Function

### PR DESCRIPTION
**Pull request by abiewardani**
Simplify Function Get String with keys

### CODE REVIEW

**Comment by antonius-hilman** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://build.bhinnekalocal.com:8080//job/goson-PR/4/

File: json_reader.go [hunk 1]
```diff
@@ -76,6 +79,19 @@ func (r *reader) GetField(keys ...string) Field {
 	return field
 }
 
+func (r *reader) GetString(key string) string {
```
Comment (agungdwiprasetyo) [inline json_reader.go:82]: klo misalnya ada case gini:
```
{
    "obj": {
        "s": "Hitomi"
    },
    "obj.s": "wkwk"
}
```
itu jadinya ngambil content yg mana yak?

klo dibikin struct `Option` aja di argument gantiin argument `key` gimana? misal:
```
type Option struct {
   Key, Separator string
}
```

**Review by agungdwiprasetyo — COMMENTED** [review]
keep code coverage 😁

File: json_reader.go (hunk 1 above)
Comment (abiewardani) [reply json_reader.go:82]: Makanya butuh separator, tadi nya mau implement di GetString cuma keknya repot yak. Mau pake option aja?

**Pull request closed without merging**